pubspec.yaml

```diff
@@ -38,24 +38,27 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.3
 
+  # Used for painting all day event widget
```

Comment:
Wait, what? Isn't dartx for utility functions and stuff?

Reply:
Generally, yes. But we only use it for the [border of the widget.]( https://github.com/student-hub/acs-upb-mobile/blob/48b200c40d09340dd744dee1b257f4ea0d8c72f3/lib/pages/timetable/view/events/all_day_event_widget.dart#L177). But I'll update the description.